lib/sedna/common/file_ops.py

```diff
@@ -290,7 +290,10 @@ def s3_download(cls, src, dst):
         import minio
         from urllib.parse import urlparse
 
-        url = urlparse(os.getenv("S3_ENDPOINT_URL", "http://s3.amazonaws.com"))
+        _url = os.getenv("S3_ENDPOINT_URL", "http://s3.amazonaws.com")
+        if not str(_url).startswith("http"):
```

Comment:
1. no str
2. explict enumerate https://, http://
```suggestion
        if not _url.startswith("http://") and not_url.startswith("https://"):
```